Logging/src/Logger.php

```diff
@@ -288,6 +277,34 @@ function (array $entry) {
      *           entry is received (with nanosecond precision). **Defaults to**
      *           the current time, generated by the client with microsecond
      *           precision.
+     *     @type string|int $severity The severity of the log entry. **Defaults to**
+     *           `"DEFAULT"`.
+     *     @type string $insertId A unique identifier for the log entry.
+     *     @type array $httpRequest Information about the HTTP request
+     *           associated with this log entry, if applicable. Please see the
+     *           [API docs](https://cloud.google.com/logging/docs/api/reference/rest/v2/LogEntry#httprequest)
+     *           for more information.
+     *     @type array $labels A set of user-defined (key, value) data that
+     *           provides additional information about the log entry.
+     *     @type array $operation Additional information about a potentially
+     *           long-running operation with which a log entry is associated.
+     *           Please see the
+     *           [API docs](https://cloud.google.com/logging/docs/api/reference/rest/v2/LogEntry#logentryoperation)
+     *           for more information.
+     *     @type string $trace Optional. Resource name of the trace associated
+     *           with the log entry, if any. If it contains a relative resource
+     *           name, the name is assumed to be relative to
+     *           //tracing.googleapis.com. Example: projects/my-projectid/traces/06796866738c859f2f19b7cfb3214824.
+     *     @type bool $traceSampled True means that the trace resource name in
+     *           the trace field was sampled for storage in a trace backend.
+     *           False means that the trace was not sampled for storage when
+     *           this log entry was written, or the sampling decision was
+     *           unknown at the time. A non-sampled trace value is still useful
+     *           as a request correlation identifier. The default is False.
+     *     @type xxx $sourceLocation Source code location information associated
```

Comment:
Does this require any changes to the `Grpc.php` implementation?

```suggestion
     *     @type array $sourceLocation Source code location information associated
```

Reply:
thank you. the placeholder to remind me to come back didn't remind me. :(